### PR TITLE
Fixup metrics for query planner

### PR DESCRIPTION
**Pull request by BrynCooke**
Names have been improved and brought into new style. number of items queued is now a real gauge.

---

**Checklist**

Complete the checklist (and note appropriate exceptions) before the PR is marked ready-for-review.

- [ ] Changes are compatible[^1]
- [ ] Documentation[^2] completed
- [ ] Performance impact assessed and acceptable
- Tests added and passing[^3]
    - [ ] Unit Tests
    - [ ] Integration Tests
    - [ ] Manual Tests

**Exceptions**

*Note any exceptions here*

**Notes**

[^1]: It may be appropriate to bring upcoming changes to the attention of other (impacted) groups. Please endeavour to do this before seeking PR approval. The mechanism for doing this will vary considerably, so use your judgement as to how and when to do this.
[^2]: Configuration is an important part of many changes. Where applicable please try to document configuration examples.
[^3]: Tick whichever testing boxes are applicable. If you are adding Manual Tests, please document the manual testing (extensively) in the Exceptions.

### CODE REVIEW

File: docs/source/configuration/telemetry/instrumentation/standard-instruments.mdx [hunk 1]
```diff
@@ -57,10 +57,14 @@ The coprocessor operations metric has the following attributes:
...
 - `apollo_router_processing_time` - Time spent processing a request (outside of waiting for external or subgraph requests) in seconds.
-- `apollo_router_query_planning_time` - Time spent planning queries in seconds.
-- `apollo_router_query_planning_warmup_duration` - Time spent planning queries in seconds.
 - `apollo_router_schema_load_duration` - Time spent loading the schema in seconds.
 
+### Query planning
+- `apollo_router.query_planning.warmup.duration` - Time spent warming up the query planner queries in seconds.
+- `apollo.router.query_planner.plan.duration` - Histogram of plan durations.
+- `apollo.router.query_planner.total.duration` - Histogram of plan durations including queue time.
+- `apollo.router.query_planner.queued` - A gauge of the number of queued plans requests.
```
Comment (garypen) [inline docs/source/configuration/telemetry/instrumentation/standard-instruments.mdx:66]: `query_planning` or `query_planner` ?

Comment (abernix) [reply docs/source/configuration/telemetry/instrumentation/standard-instruments.mdx:66]: @garypen It's suggested to me that it _should_ have been `query_planner` and that `query_planning` was a regretted decision.

Comment (BrynCooke) [reply docs/source/configuration/telemetry/instrumentation/standard-instruments.mdx]: Adjusting to `query_planning` as we now have both `query_planning` and `query_planner` keys in config.

Comment (BrynCooke) [reply docs/source/configuration/telemetry/instrumentation/standard-instruments.mdx]: In future we should try and keep verbs out of top level metric names as we typically want to include the verb on a sublevel.

**Review by abernix — APPROVED** [review]
LGTM.

**Review by Geal — APPROVED** [review]

**Pull request merged**